styleguide/_themes/global/scss/shared_assets.scss

```diff
@@ -0,0 +1,5 @@
+}
```

Comment:
I definitely like having a place to share these types of resources in the code so we don't define them multiple times.